### PR TITLE
Add workflow

### PR DESCRIPTION
**Pull request by tadakane**
## Description / Changes Made
- Create workflow file that formats frontend and backend
- Attempt to add testing to workflow  

## How to Test
1. Workflow runs automatically on Actions tab

## Checklist
- [ ] I have added/updated relevant documentation, and I have followed the coding style guidelines.
- [x] I have added/updated tests, and I have run the test suite and all tests pass.
- [x] I have checked for any potential conflicts with other branches and fixed any merge conflicts.

### CODE REVIEW

**Review by StacL — APPROVED** [review]
Ok!! 💯 :hugs:

### Description
Workflow works mainly as intended :astonished:. It hangs on a test, but team had already discussed we will work on that later.

**Review by noodaj — CHANGES_REQUESTED** [review]
Nice job on the workflow file. 
Since the workflow jest test does not work right now could you remove it.

**Review by noodaj — APPROVED** [review]

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     
+      - name: Cache npm dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Prettify frontend code
```
Comment (mitshelle) [inline .github/workflows/workflow.yml:29]: Nice! This will be helpful!

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        uses: creyD/prettier_action@v4.3
+        with:
+          # This part is also where you can pass other options, for example:
+          prettier_options: --write "**/*.{js,md,ts,tsx}" --tab-width 2 --print-width 80 --single-quote true --jsx-single-quote true --end-of-line lf
+          only_changed: True
+    
+      - name: Prettify backend code
```
Comment (mitshelle) [inline .github/workflows/workflow.yml:36]: This will help keep everything uniform!

**Review by mitshelle — APPROVED** [review]
Nice job so far @tadakane! Team has discussed that this is still in progress. Changes look good so far :)

**Pull request merged**